com.unity.render-pipelines.core/Runtime/XR/XRSystem.cs

```diff
@@ -0,0 +1,372 @@
+using System;
+using System.Collections.Generic;
+
+#if ENABLE_VR && ENABLE_XR_MODULE
+using UnityEngine.XR;
+#endif
+
+namespace UnityEngine.Rendering
+{
+    /// <summary>
+    /// Used by render pipelines to communicate with XR SDK.
+    /// </summary>
+    public static class XRSystem
+    {
+        // Valid empty pass when a camera is not using XR
+        public static readonly XRPass emptyPass = new XRPass();
```

Comment:
if this is public static, shouldnt we use k_emptyPass? I am not sure about code convention here.
also we should probably document it with <summary>

Reply:
I think for public static it's not required, but I'll add the documentation.